.github/workflows/zip-templates-and-add-to-release.yml

```diff
@@ -0,0 +1,27 @@
+name: zip-templates-and-add-to-release
```

Comment:
In other repos we use use the filename `release.yml` and the name `Release`. We should consider doing the same for this repo for consistency. It would also mean we could add other jobs to this file in future without needing to change the name.